public/app/plugins/datasource/opentsdb/components/FilterSection.tsx

```diff
@@ -138,11 +138,9 @@ export function FilterSection({
             );
           })}
         {!addFilterMode && (
-          <label className="gf-form-label query-keyword">
-            <button type="button" className={buttonStyles} onClick={changeAddFilterMode} data-testid={testIds.open}>
-              <Icon name={'plus'} />
-            </button>
-          </label>
+          <button className="gf-form-label" type="button" onClick={changeAddFilterMode} data-testid={testIds.open}>
```

Comment:
I think it's better from a11y point to use `aria-label` instead of `data-testid` here as this button doesn't have a text.